### PR TITLE
[VoiceControlManager] Add tags for missing feature and privilege

### PR DESCRIPTION
**Pull request by Inhong**
Add tags for missing feature and privilege of the below mentioned ACR.

http://suprem.sec.samsung.net/jira/browse/TCSACR-213

### CODE REVIEW

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs [hunk 1]
```diff
@@ -1943,11 +2053,8 @@ public static void StopFeedback()
         /// Called when the vc client sends audio streaming for TTS feedback.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
-        /// <privlevel>public</privlevel>
-        /// <feature>http://tizen.org/feature/speech.control</feature>
-        /// <feature>http://tizen.org/feature/microphone</feature>
-        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: event should not throw exception

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs [hunk 2]
```diff
@@ -1943,11 +2053,8 @@ public static void StopFeedback()
...
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: invalid exception for event becuase event handler have not argument

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs [hunk 3]
```diff
@@ -1840,9 +1952,13 @@ public static void SetPrivateDataProviderDelegate(PrivateDataProvider privateDat
...
         /// <since_tizen> 6 </since_tizen>
         /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
-        /// <privlevel>public</privlevel>
-        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <privilege>http://tizen.org/privilege/recorder</privilege>
+        /// <privlevel>platform</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control_manager</feature>
         /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: wrong exceptions for event

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommand.cs [hunk 4]
```diff
@@ -73,11 +73,12 @@ public class VoiceCommand : IDisposable
...
-        /// This can occur due to the following reasons:
-        /// 1) This exception can be due to an invalid state.
-        /// 2) This exception can be due to out of memory.
-        /// </exception>
+        /// <privilege> http://tizen.org/privilege/recorder </privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature> http://tizen.org/feature/microphone </feature>
+        /// <feature> http://tizen.org/feature/speech.control </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to out of memory.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommand.cs]: wrong exception becuase VoiceCommand API has not argument

Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommand.cs]: Removed ArgumentException

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs (hunk 3 above)
Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Removed ArgumentException

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs (hunk 1 above)
Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Removed InvalidOperationException

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs (hunk 2 above)
Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Removed ArgumentException

**Comment by Inhong** [comment]
Please check

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs [hunk 5]
```diff
@@ -645,10 +657,18 @@ public static State State
...
+        /// <privlevel>platform</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control_manager</feature>
         /// <feature>http://tizen.org/feature/microphone</feature>
         /// <value>The current state of the voice control service.</value>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to an invalid state.
+        /// 2) This exception can be due to operation failed.
+        /// </exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Impossible AgrumentException on readonly property

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs [hunk 6]
```diff
@@ -796,6 +821,7 @@ public static void SetCommandsFromFile(string path, CommandType type)
         /// 2) This exception can be due to operation failed.
         /// </exception>
         /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Impossible AgrumentException on this method

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs [hunk 7]
```diff
@@ -1124,9 +1168,13 @@ public static void Cancel()
...
         /// <since_tizen> 6 </since_tizen>
         /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
-        /// <privlevel>public</privlevel>
-        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <privilege>http://tizen.org/privilege/recorder</privilege>
+        /// <privlevel>platform</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control_manager</feature>
         /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```
Comment (myroot) [inline src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Impossible AgrumentException on readonly property

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs (hunk 5 above)
Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Fixed.

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs (hunk 6 above)
Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Fixed.

File: src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs (hunk 7 above)
Comment (Inhong) [reply src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs]: Fixed.

**Pull request merged**